modules/logging-loki-storage-azure.adoc

```diff
@@ -24,3 +24,23 @@ $ oc create secret generic logging-loki-azure \
   --from-literal=account_key="<azure_account_key>"
 ----
 <1> Supported environment values are `AzureGlobal`, `AzureChinaCloud`, `AzureGermanCloud`, or `AzureUSGovernment`.
+
+== Azure storage for STS enabled clusters
+If your cluster has STS enabled, the Cloud Credential Operator (CCO) supports short-term authentication using Azure AD Workload Identity. Secrets created by using CCO utility (`ccoctl`) use different format to authenticate.
```

Comment:
Same comment with respect to Azure storage secret used on LokiStack CR.

oc -n openshift-logging create secret generic logging-loki-azure-cloud-credentials \
  --from-literal=environment="AzureGlobal" \
  --from-literal=account_name="<storage_account_name>" \
  --from-literal=container="<container_name>"

Reply:
@libander This secret is not related to the secret created by CCO. It is the object storage secret that is created to provision the LokiStack CR like on [here](https://github.com/openshift/openshift-docs/pull/73846/files#diff-bb1c3db1ef6140ac250817328486db46ce8917795d4c235642761074cb3e1304L16)